src/parser/outfile.c

```diff
@@ -37,31 +43,33 @@ void    found_outfile(char **args, int i, t_cmd_info *cmd, t_list **malloc_list)
 		if (args[i + 1])
 			file_name = args[i + 1];
 		else
-			parse_error_near_nl(malloc_list);
-        truncate_outfile(file_name, cmd, malloc_list);
+			return (301);
```

Comment:
Ja cool. I started a macro list with different errorcodes on the error_msg branch. Once we merge them together, we cxould use this instead. it would then read sth like: `return (ERR_SYNTAX)` that encodes error no 301